test/functional/rpc_fundrawtransaction.py

```diff
@@ -969,6 +970,53 @@ def test_include_unsafe(self):
         signedtx = wallet.signrawtransactionwithwallet(fundedtx['hex'])
         wallet.sendrawtransaction(signedtx['hex'])
 
+    def test_inputs_min_chain_depth(self):
+        unconfirmedAddress = self.nodes[0].getnewaddress()
+        targetAddress = self.nodes[2].getnewaddress()
+        unconfirmedTxid = self.nodes[0].sendtoaddress(unconfirmedAddress, 1)
+
+        utxo1 = self.nodes[0].listunspent(0, 1, [unconfirmedAddress])[0]
+        assert unconfirmedTxid == utxo1['txid']
+
+        self.log.info("Crafting PSBT using an unconfirmed input")
```

Comment:
In 6086c58d27146d500b7ff8de89a5279a6f88457d "rpc: allow specifying min chain depth for inputs in fund calls"

```suggestion
        self.log.info("Crafting tx using an unconfirmed input")
```

Here and elsewhere in this file too.